src/LazyImage/src/Twig/BlurHashExtension.php

```diff
@@ -22,28 +22,11 @@
  */
 class BlurHashExtension extends AbstractExtension
 {
-    private $blurHash;
-
-    public function __construct(BlurHashInterface $blurHash)
-    {
-        $this->blurHash = $blurHash;
-    }
-
     public function getFunctions(): array
     {
         return [
-            new TwigFunction('data_uri_thumbnail', [$this, 'createDataUriThumbnail']),
-            new TwigFunction('blur_hash', [$this, 'blurHash']),
```

Comment:
Using a twig runtime increase performance slightly (doesn't need to load the BlurHash service on every request).

Reply:
Oh, I didn't see you add tag `twig.runtime` to `lazy_image.blur_hash`, so yeah that's logical. 

I'm usually familiar with a dedicated `<TwigExtension>Runtime`, implementing `Twig\Extension\RuntimeExtensionInterface`.

Doing things like you did does not break decoration from userland? or any custom implementation?

Reply:
> I'm usually familiar with a dedicated <TwigExtension>Runtime, implementing Twig\Extension\RuntimeExtensionInterface.

Yeah, that's just a marker interface for autoconfiguration now.

> Doing things like you did does not break decoration from userland? or any custom implementation?

Hmm, yes, I think this will be an issue if a user decides to decorate `BlurHash` as I don't believe tags are carried over to the decorated service... My gut says this isn't a realistic scenario but what do you think?

Reply:
Who knows... Before #1755 I had planned to write my own implementation for caching :D 

Well, to me that would be nice to have the possibility to decorate the default `BlurHashInterface` (e.g. add logging)

Reply:
Okay fair enough, I'll revert this change.

Reply:
Thanks :)

Reply:
Thanks :)